ui/components/ui/typography/typography.js

```diff
@@ -78,6 +78,9 @@ Typography.propTypes = {
     'h4',
     'h5',
     'h6',
+    'h7',
+    'h8',
+    'h9',
```

Comment:
Hmm... how do we have 9 different header levels? It feels like to me that you only ever need 3 header levels and after that you'd use a different name. Maybe I'm equating these to literal `<h7>`, `<h8>`, etc. elements and that's triggering something 😂 Perhaps this is more of a question for @georgewrmarshall, but this just seems a bit odd to me.

Reply:
I have removed these extra header levels I added after adjusting the subtext tags to paragraphs.